server/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog (weaverbird python package)
 
+## [0.15.0] - 2022-06-21
+
+- Feat: Add SQL backend powered by PyPika
+
```

Comment:
```suggestion
- Feat: Added a geographical hierarchy step to pandas executor
- Feat: Added a geographical dissolve step to pandas executor
- Feat: Added a geographical simplify step to pandas executor
- Feat: Support joins with geographical data in pandas executor

```

Reply:
damned!

Reply:
You can still fix it in a new PR if the tag isn't pushed yet :wink:

Reply:
My bad, there are no tags yet for the backend, so it can be fixed without trouble :smile: I think the built packages don't include the changelog anyway